v2/pkg/controller/mpi_job_controller.go

```diff
@@ -1516,6 +1567,18 @@ func (c *MPIJobController) setupSSHOnPod(podSpec *corev1.PodSpec, job *kubeflow.
 	})
 }
 
+func ownerGroupVersionKind(object metav1.Object) (*metav1.OwnerReference, schema.GroupVersionKind, error) {
```

Comment:
nit: this says nothing about owner reference that is also returned, maybe something like "getOwnerData(...)"?

Reply:
Changed to `ownerReferenceAndGVK`